docs/stencil-docs/page-builder/product-widget-tutorial.md

```diff
@@ -73,65 +57,146 @@ Accept: application/json
+
 **Response:**
 
 ```json
 {
-    "data": {
-        "uuid": "479e6ffe-73bc-4a0c-8e49-654d97691e8b",
-        "name": "Product Widget",
-        "schema": [
-            {
-                "type": "tab",
-                "label": "Content",
-                "sections": [
-                    {
-                        "label": "Product",
-                        "settings": [
-                            {
-                                "type": "productId",
-                                "label": "Product",
-                                "id": "productId",
-                                "default": "",
-                                "typeMeta": {
-                                    "placeholder": "Search by name or SKU"
-                                }
-                            }
-                        ]
-                    }
-                ]
-            }
-        ],
-        "template": "<div style=\"text-align:center\">\n<h1>{{_.data.site.product.name}}</h1>\n<div>\n<img src=\"{{_.data.site.product.defaultImage.url}}\">\n</div>\n<div>\n<p>${{_.data.site.product.prices.price.value}}</p>\n</div>\n</div>",
-        "date_created": "2020-09-08T21:40:20.310Z",
-        "date_modified": "2020-09-09T00:17:01.755Z",
-        "kind": "custom",
-        "storefront_api_query": "query Product($productId: Int = 1) { site { product(entityId: $productId) { name entityId prices { price { currencyCode value } } defaultImage { url(width: 500, height: 500) } } } } ",
-        "icon_name": "default",
-        "template_engine": "handlebars_v3",
-        "client_rerender": false,
-        "site_id": null,
-        "channel_id": 1
-    },
-    "meta": {}
+  "data": [
+    {
+      "uuid": "bb34b23b-0d4b-4b9b-9e24-c8b0dcfd5e08",
+      "template_file": "pages/category",
+      "region": "",
+      "sort_order": 0,
+      "entity_id": "21",
+      "status": "active",
+      "widget": {...},
+      ...
+  ],
+  "meta": {...}
 }
 ```
 
-## Widget preview
+<div class="HubBlock--callout">
+<div class="CalloutBlock--info">
+<div class="HubBlock-content">
 
-> You do not need to send a `POST` request to [Create a Widget](https://developer.bigcommerce.com/api-reference/store-management/widgets/widget/createwidget) or [Create a Widget Placement](https://developer.bigcommerce.com/api-reference/store-management/widgets/placement/createplacement) endpoints to reflect the new widget in Page Builder. 
+> ### Note
+> * For more information on placing and configuring widgets in the control panel, see [Page Builder](https://support.bigcommerce.com/s/article/Page-Builder) in the Help Center.
+
+</div>
+</div>
+</div>
 
-You should now see your widget with its customizable options reflected in Page Builder, under the Custom section.   
+## Place the widget using the API
```

Comment:
Although helpful, I am not sure we need to include this section in this tutorial as it covers the steps available in the existing [Widget Tutorial](https://developer.bigcommerce.com/api-docs/store-management/widgets/tutorial). Perhaps, we can improve the Widget Tutorial and link to it instead?
```suggestion
## Place the widget using the API
```

Reply:
Since the tutorial is "Create a Featured Product Widget with GraphQL", I think we should include request runners for all API requests necessary to accomplish that. That's my thought process. That way, the reader can experiment with all the example requests using the request runners without leaving the page.

Reply:
Understandable. I will leave this for Uwana to make the final determination.

Reply:
@aglensmith @ektravel. You both have valid points. I'm concerned about the maintainability of having the same information in two places. This requires updating multiple places. Is there a way to rearrange the steps in the [Widget Tutorial](https://developer.bigcommerce.com/api-docs/store-management/widgets/tutorial) so that we can request that they follow those steps first, then complete a unique last step? This would be beneficial from a maintenance perspective, and since this is the very end of this tutorial. Is the sequence of steps too different to accomplish this?